src/Symfony/Bridge/PhpUnit/bin/simple-phpunit.php

```diff
@@ -99,6 +99,7 @@
 $COMPOSER = file_exists($COMPOSER = $oldPwd.'/composer.phar')
     || ($COMPOSER = rtrim('\\' === DIRECTORY_SEPARATOR ? preg_replace('/[\r\n].*/', '', `where.exe composer.phar`) : `which composer.phar 2> /dev/null`))
     || ($COMPOSER = rtrim('\\' === DIRECTORY_SEPARATOR ? preg_replace('/[\r\n].*/', '', `where.exe composer`) : `which composer 2> /dev/null`))
+    || file_exists($COMPOSER = rtrim(`git rev-parse --show-toplevel 2> /dev/null`).DIRECTORY_SEPARATOR.'composer.phar')
```

Comment:
we should use `NUL` instead of `/dev/null` on Windows

Reply:
In dfd1f18 `NUL` is used if `'\\' === DIRECTORY_SEPARATOR`